crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/service/FullMeansAssessmentService.java

```diff
@@ -65,10 +65,8 @@ BigDecimal getAdjustedLivingAllowance(MeansAssessmentRequestDTO requestDTO, Asse
     }
 
     FullAssessmentResult getResult(BigDecimal disposableIncome, MeansAssessmentRequestDTO requestDTO, AssessmentCriteriaEntity assessmentCriteria) {
```

Comment:
We can clean up this method as requestDTO is no longer used inside this method.

Reply:
Thanks. Fixed now